wrappers/node/src/api/utils.ts

```diff
@@ -48,34 +48,6 @@ export async function provisionAgent (configAgent: string, options: IInitVCXOpti
   }
 }
 
-export async function updateAgentInfo (options: string): Promise<void> {
```

Comment:
May as well remove the mapping from `rustlib.ts`. (And maybe remove the function from other wrappers? At least let's keep in mind to do it later.)

Reply:
oh yeah touche, I've removed `vcx_agent_update_info` from node and missed this

Reply:
Removed from nodejs.

Reply:
For other ffi wrappers https://github.com/AbsaOSS/libvcx/issues/67